src/seccomp_fd_notify.c

```diff
@@ -0,0 +1,139 @@
+#include <sys/prctl.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#include <linux/audit.h>
+#include <linux/seccomp.h>
+#include <linux/filter.h>
+#include <sys/syscall.h>
+#include <stddef.h>
+
+#include <sys/socket.h>
+#include <sys/un.h>
+#include <attr/xattr.h>
+
+#include "seccomp_fd_notify.h"
+
+#define PRINT_WARNING(...)  { fprintf(stderr, __VA_ARGS__); fprintf(stderr, "\n"); }
```

Comment:
Maybe your PRINT_WARNING should just add errno?

Reply:
Hmm.. idk I don't want it printing bogus errnos from previous syscalls if the caller didn't intend.